test/extended/operators/certs.go

```diff
@@ -179,3 +200,153 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 	})
 
 })
+
+func fetchOnDiskCertificates(ctx context.Context, kubeClient kubernetes.Interface, nodeList *corev1.NodeList, testPullSpec string) (*certgraphapi.PKIList, error) {
```

Comment:
I'm +1 on making this a command as well if you like.  I think we'll eventually need it to replace the cluster-debug-tools stuff.